pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.builds

```diff
@@ -2,43 +2,10 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
-  <Import Project="$(MSBuildProjectName).props" />
-
   <ItemGroup>
-    <!-- identity project, runtime specific projects are included by props above. Will not be built on uapaot builds. -->
+    <!-- identity project, runtime specific projects are included uap.rids.props. -->
```

Comment:
I actually didn't change anything with how that part of the build works.  I can restore the comment if you like, or if you prefer that I change the way this builds I can do that too.